### ISSUE TITLE
Can not build Mask R-CNN

### ISSUE THREAD

**Issue by SharifElfouly** [NVIDIA/TensorRT#277]
## Description

I'm trying to create a TensorRT Mask R-CNN model. I'm following the steps as described [here](https://github.com/NVIDIA/TensorRT/tree/release/6.0/samples/opensource/sampleUffMaskRCNN):

When I try to run step 5: 

> 5. Apply the patch into Mask R-CNN repo to update the model from NHWC to NCHW.

The following file is missing:

`$TRT_SOURCE/samples/opensource/sampleUffMaskRCNN/converted/0001-Update-the-Mask_RCNN-model-from-NHWC-to-NCHW.patch`

There is no `/opensource` directory.

## Environment

**TensorRT Version**: 6.0.1-1
**GPU Type**: NVIDIA GTX 1070
**Nvidia Driver Version**: 
**CUDA Version**: 10.2
**CUDNN Version**: 7
**Operating System + Version**: Ubuntu 18.04.3 LTS
**Python Version (if applicable)**: 3.6.8
**TensorFlow Version (if applicable)**: 2
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:  nvcr.io/nvidia/tensorflow:19.11-tf2-py3

## Relevant Files

As described in description.

## Steps To Reproduce

Run the steps as described above in the Docker Container `nvcr.io/nvidia/tensorflow:19.11-tf2-py3`.

**Comment by chiehpower** [comment]
There are many wrong Readme.md files which were downloaded from Official website package for the latest TensorRT version7.
I saw that most README.md files are as same as TensorRT 6 version from github repo. 
For example, `running the sample` area of the README.md file in the `/TensorRT-7.0.0.11/samples/sampleUffFasterRCNN/` mentioned 
>Following the [top level guide](../../../README.md)

However, there is no any README.md file in the top level directory.
Postscript This package is `TensorRT-7.0.0.11.Ubuntu-18.04.x86_64-gnu.cuda-10.2.cudnn7.6.tar.gz`

But I knew there was the README.md file in the older version of github repo.

**Comment by rmccorm4** [comment]
Hi @SharifElfouly,

`$TRT_SOURCE` refers to the GitHub repo which you would have to clone locally, and `$TRT_RELEASE` refers to the code you download from devzone or that comes in the containers in `/usr/src/tensorrt`, `/opt/tensorrt`, `/workspace/tensorrt`, etc.

Once you clone this git repo and set TRT_SOURCE to the path where you cloned it, there is indeed an `opensource` folder: https://github.com/NVIDIA/TensorRT/samples/opensource

The samples in `$TRT_RELEASE/samples` and `$TRT_SOURCE/samples` are slightly different depending on the release. And the ~$TRT_RELEASE/samples` also comes with python samples.

**Comment by rmccorm4** [comment]
Closing, I'll reopen if you still have issues.
